frontend/src/components/GoalForm/ObjectiveFiles.js

```diff
@@ -38,50 +38,41 @@ export default function ObjectiveFiles({
 
   return (
     <>
-      {
-      readOnly && hasFiles
```

Comment:
Correct me if I'm wrong but this path doesn't seem reachable.

The return path above seems to handle all read only paths.